sdk/kusto/Azure.ResourceManager.Kusto/src/Scripts/RunTestsInRecordMode.ps1

```diff
@@ -0,0 +1,9 @@
+﻿Write-Host "Going to run test in record mode, please make sure required resources exists, if not, please run 'CreateTestResurces.ps1'" -ForegroundColor Green
```

Comment:
same as above

Reply:
moved all scripts to test path and updated readme file as agreed